File: Berita-Indo-Java/app/src/main/java/com/hafizhmo/beritaindojava/network/ApiService.java
package com.hafizhmo.beritaindojava.network;

import com.hafizhmo.beritaindojava.data.News;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface ApiService {

    //TO-DO 1: create methot 'getNews' using retrofit2.http
    @GET("{source}/terbaru")
    Call<News> getNews(@Path("source") String source);
}

File: Berita-Indo-Kotlin/app/src/main/java/com/hafizhmo/beritaindokotlin/network/ApiService.kt
package com.hafizhmo.beritaindokotlin.network

import com.hafizhmo.beritaindokotlin.data.News
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface ApiService {

    @GET("{source}/terbaru")
    fun getNews(@Path("source") source: String?): Call<News>

}